### DISCUSSION TITLE
Right way to remove some theme  colors, spacings, breakpoints...

### DISCUSSION THREAD

**Discussion by FernandoSumun** [holger1411/understrap#1774] [Q&A] [answered]
Hi all,

I'd like to know the propper way to accomplish this thing from child theme variables file, so I can get a super optimized css. For example, in a minimal styled project  I'd like to remove success, warning and danger colors and may be enough with a couple of spacing sizes. In a mobile oriented project we may want to remove some breakpoints.

Thanks in advance

**Comment by bacoords** [comment, marked as answer]
You should be able to rewrite the color maps to [remove any colors](https://getbootstrap.com/docs/5.2/customize/sass/#remove-from-map) you don't plan on using. You should also be able to choose your own [breakpoints](https://getbootstrap.com/docs/5.2/layout/breakpoints/#available-breakpoints). 

I'd also refer specifically to their documentation here: https://getbootstrap.com/docs/5.2/customize/optimize/

**Comment by FernandoSumun** [comment]
Hi,Bryan.
Thank you so much. Solved!
